courses/machine_learning/deepdive2/structured/solutions/1b_prepare_data_babyweight.ipynb

```diff
@@ -69,8 +69,8 @@
    ],
```

Comment:
<p>To ensure that the bucket will default to the project if they don't replace with their bucket name, perhaps instead it can be:</p><p>os.environ["BUCKET"] = PROJECT if BUCKET == "BUCKET" else BUCKET</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Hi Ryan,

Updated the cell command with os.environ["BUCKET"] = PROJECT if BUCKET == "BUCKET" else BUCKET.

Thanks!

Reply:
Great thanks!